HPS/Heartland/view/adminhtml/templates/form/cc.phtml

```diff
@@ -75,8 +75,10 @@ echo $code; ?>" style="display:none">
                             $("#iframes").fadeOut();                            
                             if(confirm('Are you sure you want to delete the stored card?')){
                                 $.ajax({
-                                    url: "<?php echo $block->getBaseUrl() ?>/heartland/creditcard/delete?t="+id
-                                    , success: function (data) {
+                                    url: "<?php echo $block->getBaseUrl() ?>/heartland/creditcard/delete?t="+id,
+                                    showLoader: true,
```

Comment:
@SenthilKumar-86 Will the inclusion of these loaders affect one step checkout extensions (like Magestore's)?